src/seed.js

```diff
@@ -48,11 +48,12 @@ databaseConnect()
       let product1 = await Product.create({
         name: 'Ramen',
         slug: 'ramen',
-        category: "656fcf38e9cbf8d74062e363",
-        description: "Ramen is yummy!",
+        category: '656fcf38e9cbf8d74062e363',
```

Comment:
You're hardcoding the category here.

Reply:
yeas, it should not be hardcoded, i just checked the example again. It should be `category:  category1._id`. 

I didn't check seed.js again since we were discussing yesterday that we're going to take the other approach.